plugins/processors/noise/README.md

```diff
@@ -8,6 +8,15 @@ Depending on the function, various parameters need to be configured:
 Depending on the choice of the distribution function, the respective parameters must be set. Default settings are `noise_type = "laplacian"` with `mu = 0.0` and `scale = 1.0`:
 
 ```toml @sample.conf
+The *Noise* processor is used to add noise to numerical field values. For each
+field a noise is generated using a defined probability densitiy function and
+added to the value. The function type can be configured as _Laplace_, _Gaussian_
+or _Uniform_.  Depending on the function, various parameters need to be
+configured:
+
+## Configuration
+
+```toml
```

Comment:
This doesn't look right, two ```toml stanzas in a row

Reply:
Thanks. When I merged master git automatically did this and I missed it. I just pushed a fix.